charts/xui-terms-and-conditions/requirements.yaml

```diff
@@ -3,6 +3,6 @@ dependencies:
     version: 1.9.0
     repository: '@hmctspublic'
   - name: postgresql
-    version: ~6.3.5
-    repository: '@stable'
+    version: ~8.7.0
+    repository: 'https://charts.bitnami.com/bitnami'
```

Comment:
Just making sure this line is ok :)

Reply:
yes, we not using @stable anymore as this is deprecate soon, and bitnami is what we need to use.